src/ebtree/src/ebtree.erl

```diff
@@ -411,6 +419,9 @@ range(Db, #tree{} = Tree, StartKey, EndKey, AccFun, Acc0) ->
     end).
 
 
+range(_Tx, #tree{}, #node{level = 0, members = []}, _StartKey, _EndKey, _AccFun, Acc0) ->
```

Comment:
when will a leaf node have zero members? That... should never happen.

Reply:
When the tree is empty like it says in the commit message.

Reply:
got it. can you add a test that shows the problem pls.

Reply:
also tighten this to match only the root node (instead of `level=0`)